tsctl

```diff
@@ -27,21 +29,18 @@ from flask_script import Server
 from flask_script import Option
 from flask_script import prompt_bool
 from flask_script import prompt_pass
-
 from sqlalchemy.exc import IntegrityError
+from werkzeug.exceptions import Forbidden
```

Comment:
isn't this a new dependency?

Reply:
No, this is a requirement from Flask, the web framework we are using. It has always been there.. Just a bit lower level :)